core/src/main/resources/META-INF/native-image/com.google.cloud.sql/native-image/resource-config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
nit: how about we name this directory`META-INF/native-image/com.google.cloud.sql/jdbc-socket-factory-parent/native-image.properties`? For more info: https://www.graalvm.org/22.1/reference-manual/native-image/BuildConfiguration/#embedding-a-configuration-file

Reply:
done